packages/preview/ctheorems/0.1.0/typst.toml

```diff
@@ -9,6 +9,6 @@ authors = [
     "DVDTSB",
 ]
 license = "MIT"
-description = "The classic typst-theorems package! An implementation of numbered theorem environments in Typst."
+description = "Theorem library based on (and compatible) with [typst-theorems](https://github.com/sahasatvik/typst-theorems)."
```

Comment:
Final nit: The descriptions aren't Markdown, so they can't contain links.

Reply:
fixed :D